client/tailscale/localclient.go

```diff
@@ -1514,3 +1514,12 @@ func (w *IPNBusWatcher) Next() (ipn.Notify, error) {
 	}
 	return n, nil
 }
+
+// SuggestExitNode requests an exit node suggestion and returns the exit node's details.
+func (lc *LocalClient) SuggestExitNode(ctx context.Context) (apitype.ExitNodeSuggestionResponse, error) {
```

Comment:
probably big enough of a type that I'd use a pointer here

Reply:
or would it be better to make the Location an omitempty pointer?

Reply:
discussed w/ adrian on making the response type a pointer vs. location and we agreed on making location a pointer